docs/tutorials/getting_started/moran.rst

```diff
@@ -78,14 +77,33 @@ The scores in each round::
     [7.0, 7.0, 7.0, 0.0]
     [7.0, 7.0, 7.0, 0.0]
 
+We can plot the results of a Moran process with `mp.populations_plot()`. Let's
+use a larger population to get a bit more data::
+
+   >>> import random
+   >>> import matplotlib.pyplot as plt
+   >>> axl.seed(15)  # for reproducible example
+   >>> N = 10
+   >>> players = [random.choice(axl.basic_strategies)() for i in range(N)]
+   >>> mp = axl.MoranProcess(players=players, turns=200)
+   >>> populations = mp.play()
+   >>> mp.winning_strategy_name
+   'Tit For Tat'
+    >>> ax = mp.populations_plot()
+   >>> plt.show()  #doctest: +SKIP
```

Comment:
Actually the test hangs and all the windows stay open otherwise, something to do with `plt` not exiting perhaps because the plot object isn't gc'd.